CHANGELOG.md

```diff
@@ -22,9 +22,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   incorrectly.
 - `operations_parameter_set_pairs` moved to `MemoryOrchestratorContext`
 - Ignore generated files in `docs/plugins/`
+- Treat `"~"` as the the home directory rather than a literal
 - Windows support by selecting `asyncio.ProactorEventLoop` and not using
   `asyncio.FastChildWatcher`.
-
```

Comment:
It looks like there is still a line that was deleted here

Reply:
Sorry to be picky with this, but it's important for a couple of reasons.

1. Speaking just about this file, every other section has a newline between the end of the section and the start of the next section.
2. In general, changing only the lines required for a given pull request makes is easier for people to understand what happened in any given PR or commit. Also, if you're reading the history using `git log -p` to try to figure out where something went wrong, it makes it confusing when other lines were changed and then leaves the person trying to pinpoint the bug confused as to what introduced it. For example
  - https://github.com/intel/dffml/commit/7f481f673b2d094952c0352294386dc0fdcaee84
  - https://github.com/intel/dffml/commit/070e6fb8add269af143d1914ee540ecd02fd0ae2
  - https://github.com/intel/dffml/commit/ac5cd12cc9603f2a8b216dfd051aa0019eab9a01

Reply:
I didn't find it picky it all, it's very helpful to train about the library code standards. I am on it

Reply:
Thanks! :)